### DISCUSSION TITLE
missing feature?

### DISCUSSION THREAD

**Discussion by ghost** [Bforartists/Bforartists#2701] [General]
hmm does the version on Website contain this feature? https://github.com/Bforartists/Bforartists/pull/2594

**Comment by ghost** [comment]
Wrong pull request link, updated with correct one

**Comment by ReinerBforartists** [comment]
Hm, should. Does it fail?

**Reply by ghost** [reply]
> Hm, should. Does it fail?

Hmm I hold alt but nothing happens, maybe something on my end

**Comment by ReinerBforartists** [comment]
Or a merge has killed the feature. Do you want to make a task and investigate the issue?

**Reply by ghost** [reply]
> Or a merge has killed the feature. Do you want to make a task and investigate the issue?

Sure will investigate later tonight or tomorrow
